configure.ac

```diff
@@ -86,6 +86,31 @@ AS_IF([test "x$android" = "xno"], [
   AC_DEFINE(HAVE_PROCMEM, [0], [Enable /proc/pid/mem support])
 ])
 
+# Check for termcap and readline or bypass checking for the libraries.
+AC_ARG_WITH([readline], [AS_HELP_STRING([--without-readline],
+                            [build without readline])])
+AM_CONDITIONAL([WITH_READLINE], [test "x$with_readline" != "xno"])
+AS_IF([test "x$with_readline" != "xno"], [
+  # termcap is sometimes required by readline
+  AC_CHECK_LIB([termcap], [tgetent], [], [])
+  AC_CHECK_LIB([readline], [readline], [], [
+    echo "libreadline could not be found, which is required to continue."
+    echo "The libreadline-dev package may be required."
+    exit 1
+  ])
+])
+
+GETTEXT_PACKAGE=GameConqueror
+AC_SUBST(GETTEXT_PACKAGE)
+AC_DEFINE_UNQUOTED([GETTEXT_PACKAGE], ["$GETTEXT_PACKAGE"],
+                   [The domain to use with gettext])
+
+AC_CONFIG_FILES([
+  Makefile
+  po/Makefile.in
```

Comment:
`po/Makefile.in` is created from `po/Makefile.in.in` with `intltool`. Haven't found a way not to execute `intltool` for the non-GUI part. Will try again.